app/javascript/packs/application-common.js

```diff
@@ -22,6 +22,8 @@ import { history } from '../miq-component/react-history.ts';
 import createReduxRoutingActions from '../miq-redux/redux-router-actions';
 import { formButtonsActionTypes, createFormButtonsActions } from '../forms/form-buttons-reducer';
 
+import '../../assets/stylesheets/application-webpack.scss';
```

Comment:
Should we start separating the CSS too?

(Importing `app/assets/stylesheets/` from webpack is surprising, let's not do that for any JS, but with css... 
¯\\\_(ツ)\_/¯ up to you :))

Reply:
Ideally we would not need any custom CSS :trollface: but I get your point. Maybe `/app/stylesheets`?

Reply:
Works for me :) Thanks :)

(if you want to be 100% consistent, `app/stylesheet`, because asset pipeline has plurals but webpack has `app/javascript`. But both are fine :).)

Reply:
> Ideally we would not need any custom CSS :trollface: but I get your point. Maybe /app/stylesheets? 

That ship has already sailed...

Reply:
`/app/stylesheet` it is then